frontend/lib/analytics/google-analytics.tsx

```diff
@@ -283,14 +283,23 @@ type OutboundLinkProps = DetailedHTMLProps<
 > & {
   /** The "href" prop is required on outbound links, not optional. */
   href: string;
+  target: string;
+  rel: string;
 };
 
+const defaultOutboundLinkProps = {
+  target: "_blank",
+  rel: "noopener noreferrer"
+}
 /**
  * A react component that encapsulates a link to an external website,
  * which we want to track with analytics.
  */
 export function OutboundLink(props: OutboundLinkProps): JSX.Element {
-  const { onClick, ...otherProps } = props;
+  console.log("before default", props);
+  props = { ...props, ...defaultOutboundLinkProps,};
+  console.log("after default", props);
```

Comment:
Oh also be sure to remove these `console.log`'s before we merge this into the codebase--it's great to do this while developing a feature, but once it's done the lines ought to be removed so they don't spam the console for other developers working on different features.

Reply:
Definitely